src/coins.cpp

```diff
@@ -381,10 +379,12 @@ static bool ExecuteBackedWrapper(Func func, const std::vector<std::function<void
     }
 }
 
-bool CCoinsViewErrorCatcher::GetCoin(const COutPoint &outpoint, Coin &coin) const {
-    return ExecuteBackedWrapper([&]() { return CCoinsViewBacked::GetCoin(outpoint, coin); }, m_err_callbacks);
+std::optional<Coin> CCoinsViewErrorCatcher::GetCoin(const COutPoint& outpoint) const
+{
+    return ExecuteBackedWrapper<std::optional<Coin>>([&]() { return CCoinsViewBacked::GetCoin(outpoint); }, m_err_callbacks);
 }
 
-bool CCoinsViewErrorCatcher::HaveCoin(const COutPoint &outpoint) const {
-    return ExecuteBackedWrapper([&]() { return CCoinsViewBacked::HaveCoin(outpoint); }, m_err_callbacks);
+bool CCoinsViewErrorCatcher::HaveCoin(const COutPoint &outpoint) const
```

Comment:
Hmm looks like you moved the `&` now too, along with the brace on the newline.

Reply:
Yes, formatted the cpp, since it was modified, reverted the header, since it wasn't.

Resolving this comment, as the developer notes state:
> However, we're now trying to converge to a single style, which is specified below. When writing patches, favor the new style over attempting to mimic the surrounding style, except for move-only commits.

But let me know if it was correct before, and my modification is incorrect, maybe I misunderstood what you meant.

Reply:
I meant that the `HaveCoin` signature is not relevant to this PR. So, just modifying it by moving the brace to the next line and moving the `&` to be next to the type is making a style change to a line that doesn't need to be touched in this PR.

If a line must be changed for the PR anyways, then yes I agree the style of the entire line should be updated.